analytics-webapps/src/main/webapp/vue-app/generic-portlet/components/chart/AnalyticsChart.vue

```diff
@@ -122,6 +130,9 @@ export default {
           trigger: 'item',
           formatter: '{b} : {c} ({d}%)'
         };
+        chartOptions.legend = {
+          bottom: '0%'
+        };
```

Comment:
I think it's better to place labels in top left in order to have more spaces when having multiple labels.
Or we can as well make it configurable in settings UI, to check with @mzorai 
```suggestion
        chartOptions.legend = {
          orient: 'vertical',
          left: 'left',
        };
```